src/uct/ib/ud/base/ud_ep.c

```diff
@@ -712,7 +713,7 @@ ucs_status_t uct_ud_ep_flush_nolock(uct_ud_iface_t *iface, uct_ud_ep_t *ep,
             /* Add dummy skb to the window, which would call user completion
              * callback when getting ACK.
              */
-            skb->flags                 |= UCT_UD_SEND_SKB_FLAG_COMP;
+            skb->flags                  = UCT_UD_SEND_SKB_FLAG_COMP;
             skb->len                    = sizeof(skb->neth[0]);
```

Comment:
because skb was just taken from the pool, valgring shows an error about using of uninitialized memory